### DISCUSSION TITLE
Token expiration

### DISCUSSION THREAD

**Discussion by Xav-74** [persandstrom/python-verisure#165] [General]
Hi @persandstrom 

First, thanks for your help. I develop a plugin Verisure for Jeedom system.
As I'm based in France, I don't have the same equipment like you. But thanks to your code, I rewrite the API connector in php for Belgium users.
https://github.com/Xav-74/verisure/tree/beta/3rdparty  (type 2 in this case, type 1 is for french model).

I only have a problem with the token. It seems to be valid for 55 days. But in fact, I have a 401 error (Unauthorized) after about 3 days. And I need to do a new 2FA authentication. Do you have the same issue ?
I’m not a professional developer so the problem is probably me 😊

Thanks 

Xav

**Comment by persandstrom** [comment]
Hi,

I have not been using 2FA for a while now. So I am  not sure.

I guess you could execute the the CLI of python-verisue using the Linux watch command for three days and see if it acts differently.

**Comment by Xav-74** [comment]
Hi 

That a great idea! I will check that.
I thought the 2FA was mandatory. I'll try to deactivate this feature. More simple maybe :)

Thanks a lot for your help!
